spec/models/event_spec.rb

```diff
@@ -8,7 +8,6 @@
     expect(event.errors[:description]).to_not be_empty
     expect(event.errors[:website]).to_not be_empty
     expect(event.errors[:start_at]).to_not be_empty
-    expect(event.errors[:end_at]).to_not be_empty
     expect(event.errors[:event_type]).to_not be_empty
```

Comment:
We're not testing that rails validations work.  We're testing that the specific validations we want on the model are present.  We don't own the Rails validation code, but we own the validations we implement.

They're valuable because they protect us against accidental changes to the production code.  If some future dev comments out or deletes a line, then we want our test suite to raise a red flag.